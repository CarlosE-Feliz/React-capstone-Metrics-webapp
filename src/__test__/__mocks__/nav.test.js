import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router, Link } from 'react-router-dom';

test('renders correctly', () => {
  const tree = renderer
    .create(
      <Router basename={process.env.PUBLIC_URL}>
        <Link to="/" active="true" exact="true">
          <Link id="back" className="back" to="/" exact> &#8249;</Link>
        </Link>
      </Router>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
