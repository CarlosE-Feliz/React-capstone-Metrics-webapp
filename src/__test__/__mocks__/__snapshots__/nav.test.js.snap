// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  active="true"
  exact="true"
  href="/"
  onClick={[Function]}
>
  <a
    className="back"
    exact={true}
    href="/"
    id="back"
    onClick={[Function]}
  >
     â€¹
  </a>
</a>
`;
